index.html

```diff
@@ -73,8 +73,638 @@
+                <div class="img-box">
+                    <a href="#"><img class="feat-img" src="./images/biden2.png" alt=""/></a>
+                  <div class="bottom-left">
+                    <p><a class="text-danger fw-bold text-decoration-none bg-grey p-2" href="#">POLITICS</a></p>
+                  </div>
+                </div>
+                <div class="img-text">
+                  <h5>
+                    <a class="fw-bold text-decoration-none text-dark" href="#"
+                      >Joe Biden's First Executive Actions Undo Many of Donald
+                      Trump's</a
+                    >
+                  </h5>
+                  <p>
+                    Actions related to the COVID-19 crisis, climate change and
+                    racial equality are to be among the first taken as he begins his
+                    tenure.
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div class="p-2 mb-2" id="my-turn">
+              <section>
+                <h6 class="text-secondary">MY TURN</h6>
+                <div>
+                  <div class="myturn d-md-flex justify-n">
+                    <div>
+                      <a href="#" class="text-decoration-none">
+                        <img class="l-img" src="./images/myturn1.jpg" alt="" />
+                        <p class="fw-bold text-secondary">'I've Rescued Hundreds of Special Needs Dogs'</p>
+                      </a>
+                    </div>
+                    <div>
+                      <a href="#" class="text-decoration-none">
+                        <img class="l-img" src="./images/myturn1.jpg" alt="" />
+                        <p class="fw-bold text-secondary">'I've Rescued Hundreds of Special Needs Dogs'</p>
+                      </a>
+                    </div>
+                    <div>
+                      <a href="#" class="text-decoration-none">
+                        <img class="l-img" src="./images/myturn1.jpg" alt="" />
+                        <p class="fw-bold text-secondary">'I've Rescued Hundreds of Special Needs Dogs'</p>
+                      </a>
+                    </div>
+                  </div>
+                </div>
+              </section>
+            </div>
+            <div class="p-2 mb-2" id="culture">
+              <section>
+                <h6 class="text-secondary">CULTURE & TRAVEL</h6>
+                <div class="culture d-md-flex justify-n">
+                  <div>
+                    <a href="#" class="text-decoration-none">
+                      <div class="img-box">
+                        <img class="l-img" src="./images/culture2.png" alt="" />
+                        <div class="bottom-left">
+                        <p class="bottom-left text-dark fw-bold bg-light-sky">Music</p>
+                        </div>
+                      </div>
+                        <p class="fw-bold text-secondary m-2">These Are the Top 100 Songs of 2020</p>

```

Comment:
- [ ] Every title in this category should be an `h` tag. Please use an `h` as well for other titles in this section.